stable/atlantis/templates/statefulset.yaml

```diff
@@ -58,10 +67,23 @@ spec:
           {{- if .Values.allowRepoConfig }}
             - --allow-repo-config
           {{- end }}
+          {{- if .Values.logLevel }}
+            - --log-level={{ .Values.logLevel }}
```

Comment:
Why not use the environment variable for this instead? Obviously there's no real difference between the flag and the env var but maybe best to be consistent?

Reply:
Any reason we don't do this for other specific values like allowRepoConfig?  Where do we decide the difference?  Does it matter?

Reply:
I think he probably used the flag for `--allow-repo-config` because it was a boolean. But you can set the `values` for it to `false` and then just use `ATLANTIS_ALLOW_REPO_CONFIG={{.Values.allowRepoConfig}}`.

Reply:
If it's set to the string "false" it's interpreted as false.

https://github.com/runatlantis/atlantis/blob/master/cmd/server_test.go#L570
https://github.com/runatlantis/atlantis/blob/master/cmd/server_test.go#L601

Reply:
I like having it as a first-class configuration but then converting it into the environment variable